README.md

```
@@ -80,10 +80,11 @@ This license grants you the freedom to use flash-kv in your own projects, under
 
 
 [Github-url]: https://github.com/KevinZh0C/FlashKV-rs
-[CI-url]: https://github.com/KevinZh0C/FlashKV-rs/actions/workflows/rust.yml
+[CI-url]: https://github.com/Kevinzh0C/FlashKV-rs/actions/workflows/rust.yml
 [doc-url]: https://docs.rs/flash-kv
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "doc-url"

(MD053, link-image-reference-definitions)